documentation/bolt_configuration_reference.md.erb

```diff
@@ -121,11 +121,19 @@ Transport configuration options can be set in both the configuration file and in
 <% @transports[:options].each do |transport, options| %>
 ### <%= transport %>
 
+<% if transport == 'ssh' %>
+| Option | Description | External SSH | Type | Default |
+| ------ | ----------- | ------------ | ---- | ------- |
```

Comment:
```suggestion
| ------ | ----------- | :----------: | ---- | ------- |
```

Reply:
@beechtom Is there a better name for the column? I don't love external ssh...

Reply:
@hestonhoffman Might also have some ideas ^

Reply:
I don't mind external ssh... if anything, maybe a small blurb before the options saying that not all options are compatible when using external ssh?